libDCM/hilsim.c

```diff
@@ -21,10 +21,121 @@
 

```

Comment:
I would see this file as not being part of libDCM which in my mind is the IMU portion of MatrixPilot. How about putting it back libUDB if it has to go somewhere more mainstream than in Tools/....